blog-samples/CSharp/FacebookHandover/Secondary/Bots/SecondaryBot.cs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+//
+// Generated with Bot Builder V4 SDK Template for Visual Studio EchoBot v4.3.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FacebookModel;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs.Choices;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Secondary.Bots
+{
+    public class SecondaryBot : ActivityHandler
+    {
+        private const string OPTION_PASS_PRIMARY_BOT = "Pass to primary";
+        private const string OPTION_REQUEST_THREAD_CONTROL = "Receive request";
+        private const string OPTION_REQUEST_THREAD_CONTROL_NICELY = "Receive nice request";
+        private const string OPTION_TAKE_THREAD_CONTROL = "Have control taken";
+
+        private static readonly string[] _options = new[] { OPTION_PASS_PRIMARY_BOT, OPTION_TAKE_THREAD_CONTROL };
```

Comment:
This could be a static array of Choice, since that is the only thing this is used for.